src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -8,11 +8,11 @@
             [oops.core :as oops]
             [react-native.background-timer :as background-timer]
             [status-im2.common.constants :as constants]
+            [quo2.components.dividers.date :as divider-date]
```

Comment:
please export your component from quo2/core: https://github.com/status-im/status-mobile/blob/develop/src/quo2/core.cljs#L94

and then import this as `[quo2.core :as quo]` so it can be used as 
`quo/divider-date`

Reply:
Done